### ISSUE TITLE
[BUG] Panon visualizer doesn't react to audio

### ISSUE THREAD

**Issue by emmalexandria** [rbn42/panon#86]
**Desktop**
 - OS: Arch Linux
 - KDE Frameworks Version: 5.91.0

**Describe the bug**
No matter what audio I play, Panon doesn't react to playing audio, it just stays flat. Tried all audio backends and multiple styles.
![image](https://user-images.githubusercontent.com/68666531/154838307-758a24c4-268f-4ede-980a-04e0cb40ff58.png)

**Any error message shown in the console**
```
QML debugging is enabled. Only use this in a safe environment.
Icon theme "elementary" not found.
qml: View QML loaded
kf.plasma.quick: Applet preload policy set to 1
Toolbox not loading, toolbox package is either invalid or disabled.
qml: New Containment: ContainmentInterface(0x5653bf639900)
QQuickOpenGLShaderEffect: 'source' does not have a matching property!
QQuickOpenGLShaderEffect: 'source' does not have a matching property!
trying to show an empty dialog
kf.kio.core: Invalid URL: QUrl("")
Trying to take space not available BasicAppletContainer_QMLTYPE_148_QML_165(0x5653bf636f50, parent=0x5653bf65fd20, geometry=320,240 640x96)
file:///home/emma/src/panon/plasmoid/contents/ui/Spectrum.qml:95:45: QML Image: Unknown error
qml:
qml:
Trying to take space not available BasicAppletContainer_QMLTYPE_148_QML_165(0x5653bf636f50, parent=0x5653bf65fd20, geometry=320,240 640x96)
trying to show an empty dialog

```

Installed both through ocs-url and AUR, both had same issue.

**Comment by Aoshisen** [comment]
I got some error in my OS

**Comment by ViejoRS** [comment]
Edit: The solution here fixed the problem for me - https://github.com/rbn42/panon/issues/78#issuecomment-1007282512

I have the same error on both Arch and EndeavourOS.
When I was on Kubuntu or Debian, Panon worked appropriately.

I'm not sure what the difference is, I've confirmed I have all required dependencies. Similar to OP, I've tried all backends and input settings. I've also tried with both pipewire-pulse and with standard pulseaudio installed, and there is also no difference between X11 and Wayland environments.

**Comment by mrlpm** [comment]
@ViejoRS Same error here in Arch recent install

**Comment by cangkui** [comment]
Yes, I am meeting a similar problem like this. Actually when I change the audio data source to "PortAudio", panon works properly and it catches my microphone signals and make them visual. When I change the audio data source to "PulseAudio" and it doesnt work properly in every mode.

**Comment by cangkui** [comment]
> Yes, I am meeting a similar problem like this. Actually when I change the audio data source to "PortAudio", panon works properly and it catches my microphone signals and make them visual. When I change the audio data source to "PulseAudio" and it doesnt work properly in every mode.

Arch 6.1.9 with KDE Plasma 5.26.5, X11.

**Comment by MurzNN** [comment]
The same problem on Ubuntu 23.04, it uses pipewire-pulse and with "PortAudio" panon shows only microphone signal, not the speakers signal, and I see no ways to choose the input channel manually.

I've created a feature request about this: https://github.com/rbn42/panon/issues/112
